### DISCUSSION TITLE
Possible to modify AbsorptionIndirectSteam to allow hot water applications?

### DISCUSSION THREAD

**Discussion by leodaniel** [lbl-srg/modelica-buildings#3816] [Q&A] [answered]
The Modelica Buildings Library allows the simulation of Steam powered absorption chiller `Buildings.Fluid.Chillers.AbsorptionIndirectSteam`. But will it be possible to modify the same for hot water powered absorption chiller? 

Please provide suggestions.

**Comment by mwetter** [comment, marked as answer]
If you can present your equipment with the used performance curves, it should be fine to use that model.

**Reply by leodaniel** [reply]
Thank Dr. Wetter.
